parent/pom.xml

```diff
@@ -59,6 +57,13 @@
         <yumArtifactName>emodb-web-${project.version}</yumArtifactName>
         <yumArtifactId>emodb-web</yumArtifactId>
 
+        <jacoco.version>0.8.6</jacoco.version>
```

Comment:
Sorry for nitpicking, but it seems like an agreement in parent pom.xml: properties which set versions of artifacts or plugins go first.

Reply:
don't think we have pattern, everything is mixed.
i've just made separate sections for jacoco and sonar for better visibility.